GenderPayGap.WebUI/Views/Register/ReviewAddress.cshtml

```diff
@@ -102,43 +119,128 @@
-
-                    @await Html.PartialAsync("DisplayReference", new {name = "Mutual partnership number", value = Model.MutualNumber, defaultValue = ""})
-
-                    @await Html.PartialAsync("DisplayReference", new {name = Model.OtherName, value = Model.OtherValue, defaultValue = ""})
-
+                    @if (!string.IsNullOrWhiteSpace(Model.CompanyNumber))
+                    {
+                        <tr class="govuk-table__row">
+                            <th class="govuk-table__header" scope="row">
+                                Company Number
+                            </th>
+                            <td class="govuk-table__cell">
+                                @Model.CompanyNumber
+                            </td>
+                        </tr>
+                    }
+                    
+                    @if (!string.IsNullOrWhiteSpace(Model.CharityNumber))
+                    {
+                        <tr class="govuk-table__row">
+                            <th class="govuk-table__header" scope="row">
+                                Charity Number
+                            </th>
+                            <td class="govuk-table__cell">
+                                @Model.CharityNumber
+                            </td>
+                        </tr>
+                    }
+                    
+                    @if (!string.IsNullOrWhiteSpace(Model.CharityNumber))
+                    {
+                        <tr class="govuk-table__row">
+                            <th class="govuk-table__header" scope="row">
+                                Mutual partnership number
+                            </th>
+                            <td class="govuk-table__cell">
+                                @Model.MutualNumber
+                            </td>
+                        </tr>
+                    }
+                    
+                    @if (!string.IsNullOrWhiteSpace(Model.CharityNumber))
+                    {
+                        <tr class="govuk-table__row">
+                            <th class="govuk-table__header" scope="row">
+                                @Model.OtherName
+                            </th>
+                            <td class="govuk-table__cell">
+                                @Model.OtherValue
+                            </td>
+                        </tr>
+                    }
                 </tbody>
             </table>
-
-            <fieldset>
-                <br>
-                <p>
-                    <button type="submit" class="button" name="command" value="approve">Confirm and complete registration</button>
-                </p>
-                <p>
-                    <button type="submit" class="link-button" name="command" value="decline">Reject registration</button>
-                </p>
-            </fieldset>
-        </div>
+            
+        <h2 class="govuk-heading-l">

```

Comment:
The "similar organisations" section is two-thirds width, which I feel is a bit too narrow (it's fine for the above info but makes the table awfully squashed)